pypeman/tests/test_channel.py

```diff
@@ -234,6 +241,28 @@ def test_final_nodes(self):
             vars(self.clean_msg(msg1)), vars(self.clean_msg(endnode_input)),
             "Channel final_nodes don't takes event msg in input")
 
+    def test_init_nodes(self):
+        """ Whether BaseChannel init_nodes is working """
+        chan1 = BaseChannel(name="test_channel_init_clbk", loop=self.loop)
+        initouttext = "tutu"
```

Comment:
why tutu and not just an empty text or "unspecified" r "default"?
Is this a relict from debugging?

Reply:
Yup relict from debug that i didnt change, changed to inittxt